hack/generator/pkg/codegen/pipeline_export_generated_code.go

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
kicking ideas around here -- wondering if these pipeline stages should be in a sub-module (i.e. `pipeline` inside `codegen` or something)?

Reply:
I tried moving them into a submodule, but ended up with a circular import error.